tensorboard/plugins/projector/polymer3/vz_projector/vz-projector-bookmark-panel.ts

```diff
@@ -40,7 +40,7 @@ export class BookmarkPanel extends LegacyElementMixin(PolymerElement) {
   @property({type: Number})
   selectedState: number;
 
-  private projector: Projector;
+  private projector: any;
```

Comment:
Might be nice to keep around the types as comments (`// actual type: Projector`) but up to you since it's extra effort already.